src/validation.cpp

```diff
@@ -5023,12 +5024,19 @@ bool LoadMempool(CTxMemPool& pool)
         for (const auto& i : mapDeltas) {
             pool.PrioritiseTransaction(i.first, i.second);
         }
+
+        {
+            LOCK(pool.cs);
+            file >> pool.m_unbroadcast_txids;
```

Comment:
6307582b4d30167a931678d12e99a00a0e75e81a

This is not safe. You are erasing the existing content of the `m_unbroadcast_txids`

Reply:
ah, thanks for catching. fixed via new mempool methods